src/control-center/src/main.cpp

```diff
@@ -1,32 +1,55 @@
 #include <iostream>
 #include <ControlCenter.h>
 #include <Serializer.h>
+#include <Logger.h>
+#include <ConfigReader.h>
 #include <csignal>
 #include <chrono>
 #include <thread>
 
 using namespace std;
 
-Serializer s;
-ControlCenter cc(s);
-bool quit = false;
+namespace {
+    static const int8_t BROADCAST_SENSORS_INTERVAL = 3;
+    ControlCenter *g_cc;
+}
+
+volatile bool quit = false;
 
 void signal_handler( int signum ) {
-    cout << "Received: " << signum << endl;
-    cc.close_connection();
-    cout << "Closed socket..." << endl;
+    log() << "Received: " << signum;
+    g_cc->close_connection();
     quit = true;
     exit(signum);
```

Comment:
Jeśli już jest flaga `quit` to oczekiwałbym "gracefull-exit", a nie brutalne ubicie programu - obecnie `quit` jest efektywnie bezużyteczne.